frontend/src/components/Profile/ProfileImage.js

```diff
@@ -1,37 +1,97 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { Box, Typography } from '@mui/material';
 import Button from '@mui/material/Button';
 import useProfileImage from './useProfileImage';
 import uploadProfileImage from '../../requests/put/uploadProfileImage';
 import Alert from '@mui/material/Alert';
+import deleteProfileImage from '../../requests/delete/deleteProfileImage';
+import Modal from '@mui/material/Modal';
+import styles from './styles';
 
 function ProfileImage() {
 
     let profileImage = useProfileImage();
     const [uploadedFile, setUploadedFile] = useState(null);
-    const [successAlert, setSuccessAlert] = useState(false);
+    const [successAlert, setSuccessAlert] = useState(null);
     const [errorAlert, setErrorAlert] = useState(null);
+    const [modalOpen, setModalOpen] = useState(false);
+    const [disabled, setDisabled] = useState(false);
 
     const handleFileChange = (event) => {
         const selectedFile = event.target.files[0];
         setUploadedFile(selectedFile);
     };
 
+    const handleFileDelete = () => {
+        setDisabled(true);
+        deleteProfileImage()
+            .then(res => {
+                setSuccessAlert(res.message);
+                setErrorAlert(null);
+
+                setUploadedFile(null);
+            })
+            .catch(err => {
+                setErrorAlert(err.message);
+                setSuccessAlert(false);
+            })
+            .finally(() => {
+                setModalOpen(false);
+                setDisabled(false);
+            })
+    };
+
     const handleFileUpload = () => {
         uploadProfileImage(uploadedFile)
-        .then(res => {
-            setSuccessAlert(res.message);
-            setErrorAlert(null);
-        })
-        .catch(err => {
-            setErrorAlert(err.message);
-            setSuccessAlert(false);
-        })
-    };        
+            .then(res => {
+                setSuccessAlert(res.message);
+                setErrorAlert(null);
+            })
+            .catch(err => {
+                setErrorAlert(err.message);
+                setSuccessAlert(false);
+            })
+            .finally(() => {
```

Comment:
Why do we need this block of code (finally part)? I think it should work without it, but if we need it, we need to change it. In react we don't use document object, we don't directly manipulate the DOM using document methods. Instead we use React's state management.

If we really need finally block of code to set image value to null, use something like this (not exactly like this but just to give you an idea):

const [imageValue, setImageValue] = useState('');

...

<input
  type="text"
  id="image"
  value={imageValue}
  onChange={e => setImageValue(e.target.value)}
/>